docs/topics/cache.txt

```diff
@@ -401,6 +403,19 @@ behavior. These arguments are provided as additional keys in the
     On some backends (``database`` in particular) this makes culling *much*
     faster at the expense of more cache misses.
 
+  .. versionchanged:: 1.11
```

Comment:
This note should be self-contained as described in https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features